### ISSUE TITLE
Look at running out of memory problem for strong scaling jobs

### ISSUE THREAD

**Issue by lyon-fnal** [lyon-fnal/IRMA#19]
The 1x32 job failed with out of memory. Try to figure out why.

**Comment by lyon-fnal** [comment]
Trying to run vTune. NERSC documentation is at https://docs.nersc.gov/development/performance-debugging-tools/vtune/
Ok - can't get VTune to work correctly on my Mac (it segfaults when it tries to read a vtune file that I copied over from Cori). Trying Arm Forge Map

**Comment by lyon-fnal** [comment]
MAP doesn't work either. Julia blows up on dynamic link loading (I think where MAP is trying to inject its library).

**Comment by lyon-fnal** [comment]
Maybe go back to VTune and try to run over X11 ... with NOMachine.

**Comment by lyon-fnal** [comment]
ok - I have `jobs/019_OutOfMemory` and see `vtune_mem` within. I ran the VTune gui (over NOMachine). I'm not sure if I believe what I got...

![image](https://user-images.githubusercontent.com/9458395/97039185-a3c35380-1531-11eb-99c0-a57977472a7b.png)

25 MB doesn't seem right and I don't see any mention of anything regarding Julia. 

There are a lot of warnings,
![image](https://user-images.githubusercontent.com/9458395/97039237-b63d8d00-1531-11eb-9340-53e668abc9b5.png)

Again, no reference to Julia. The files that it can't find do in fact exist at the specified location. So something strange is going on.
